### PR TITLE
Added functional for write metadata of foundtion to file

### PR DESCRIPTION
**Pull request by 1m8o5s**
In result metadata file looks like:

image
sorry for image, github was write text without tabs

For find length and area of polygon I get points from facets of top and add they to polygon

Method writeMetadata accepts argument foundationMinimalZCoord and copy of original foundation, because after integrating with input model, in case when we get console argument -b, foundation contains input model too and hard to take other parameters of foundation

### CODE REVIEW

File: FoundationBuilder/include/MetadataOutput.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <string>
+
+namespace geom_utils {
+	class Mesh;
+	void writeMetadata(const Mesh& inputModel, const Mesh& foundation, const std::string& metadataFile, const std::string& inputModelName, const std::string& foundationName, const bool& buildIn, const float& timeOfConstructing, const float& foundationMinimalZCoord);
```
Comment (VladGribinchuk) [inline FoundationBuilder/include/MetadataOutput.h:6]: Please prefer passing simple types like bool, float, etc. by value.
Passing by const ref makes sense if argument is non-trivial type with non-trivial copy ctor, like our Mesh type. It has 'heavy' copy ctor, so we have some profit when passing it by const ref without copying. The same cant be said about simple types.

**Review by VladGribinchuk — COMMENTED** [review]
Good, but please see one minor note

**Review by VladGribinchuk — APPROVED** [review]
..but that my remark is not critical, it is just FYI. So, lets merge it, if you dont mind)

**Comment by 1m8o5s** [comment]
I fail with commits and dont know how to fix it

**Comment by VladGribinchuk** [comment]
Whats the problem?

**Comment by 1m8o5s** [comment]
In commit message I write not correct message and try to change it by revert

**Comment by VladGribinchuk** [comment]
There is `git commit --amend` command for changing commit message

**Comment by 1m8o5s** [comment]
> There is `git commit --amend` command for changing commit message

thanks, i will remember

**Comment by 1m8o5s** [comment]
I will create new pull request for fix this

**Pull request merged**